src/hints/patricia.rs

```diff
@@ -22,29 +22,27 @@ use crate::cairo_types::trie::NodeEdge;
 use crate::hints::types::{skip_verification_if_configured, PatriciaSkipValidationRunner, Preimage};
 use crate::hints::vars;
 use crate::starknet::starknet_storage::StorageLeaf;
-use crate::starkware_utils::commitment_tree::base_types::{DescentMap, DescentStart, Height, NodePath};
+use crate::starkware_utils::commitment_tree::base_types::{DescentMap, DescentPath, DescentStart, Height, NodePath};
 use crate::starkware_utils::commitment_tree::patricia_tree::patricia_guess_descents::patricia_guess_descents;
 use crate::starkware_utils::commitment_tree::update_tree::{
     build_update_tree, decode_node, DecodeNodeCase, DecodedNode, TreeUpdate,
 };
 
 pub const SET_SIBLINGS: &str = "memory[ids.siblings], ids.word = descend";
 
-pub type Descend = (Felt252, Felt252);
-
 pub fn set_siblings(
     vm: &mut VirtualMachine,
     exec_scopes: &mut ExecutionScopes,
     ids_data: &HashMap<String, HintReference>,
     ap_tracking: &ApTracking,
     _constants: &HashMap<String, Felt252>,
 ) -> Result<(), HintError> {
-    let descend: Descend = exec_scopes.get(vars::scopes::DESCEND)?;
+    let descend: DescentPath = exec_scopes.get(vars::scopes::DESCEND)?;
 
     let siblings = get_ptr_from_var_name(vars::ids::SIBLINGS, vm, ids_data, ap_tracking)?;
-    vm.insert_value(siblings, descend.0)?;
+    vm.insert_value(siblings, Felt252::from(descend.0.0))?;
```

Comment:
Declare a variable `length` and `word` like it is done at line 175-176 for documentation purposes?

Reply:
Done!